platforms/windows/SQLitePlugin/SQLitePlugin.cpp

```diff
@@ -0,0 +1,511 @@
+#include "pch.h"
+
+#include "SQLitePlugin.h"
+
+using namespace winrt;
+using namespace winrt::Microsoft::ReactNative;
+using namespace winrt::Windows::Foundation;
+using namespace winrt::Windows::Storage;
+using namespace winrt::Windows::Storage::Streams;
+using namespace winrt::Windows::Security::Cryptography;
+
+namespace SQLitePlugin
+{
+    void SQLitePlugin::Attach(
```

Comment:
what does this method do?

Reply:
I did some research, and it seems like this method sort of "mounts" another database file. In theory, it could be executed as a regular SQL query, but you need to specify a path to the DB file, which react app might not be able to resolve. 

The C# implementation lacked support for this method, but since it is present on iOS and Android, I will add it